wasm/interpreter/interpreter.go

```diff
@@ -477,12 +477,12 @@ func (it *interpreter) Call(f *wasm.FunctionInstance, params ...uint64) (results
 		return
 	}
 
+	for _, param := range params {
+		it.push(param)
+	}
 	if g.hostFn != nil {
 		it.callHostFunc(g, params...)
```

Comment:
Could you delete `_ ...uint64` from `callHostFunc`'s signature? https://github.com/tetratelabs/wazero/blob/07248537ca015313b4dcc16e2507f4905c8eac91/wasm/interpreter/interpreter.go#L495

Reply:
sur